config/zones/MX-CE.yaml

```
@@ -4,3 +4,5 @@ bounding_box:
   - - -96.22812130395499
     - 21.325344143417453
 timezone: null
```

Should we add the timezone here?
```suggestion
timezone: America/Tijuana
```